### DISCUSSION TITLE
assignments finished/submit notification

### DISCUSSION THREAD

**Discussion by bhuffaker** [community/community#17883] [GitHub Education]
I have rolling assignments where students can submit projects at different times.  Currently there is no way for a student to "submit" or mark as "finished" an assignment. 

Please add "submit"/"finished"/"ready for review" button to the assignments page so I know when to review the code and can see from the assignments page which assignments have been submitted.

**Comment by ssyberg** [comment]
I opened a [similar ticket](https://github.com/orgs/community/discussions/31488), as the concept of submissions appears in the interface, I just don't see a way to use it.

**Comment by bhuffaker** [comment]
I think they want the student to add a comment to the pull request, but I don’t see how that would feed back into submissions.

**Reply by ssyberg** [reply]
Did you see this documented somewhere? What format should the comment take?

**Comment by bhuffaker** [comment]
Not really, but the email generated when the assignment is cloned does have some documentation. 
I would recommend creating a test assignment and then then clone it, so you can see it.

**Reply by ssyberg** [reply]
> Not really, but the email generated when the assignment is cloned does have some documentation. I would recommend creating a test assignment and then then clone it, so you can see it.
> […](#)
> On Sep 1, 2022, at 5:22 AM, Seth Syberg ***@***.***> wrote: Did you see this documented somewhere? What format should the comment take? — Reply to this email directly, view it on GitHub <https://github.com/orgs/community/discussions/17883#discussioncomment-3526490>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AECPT7MWSTEOX6XWGKI6YRLV4CNZ5ANCNFSM5XZXQKFA>. You are receiving this because you authored the thread.

Hmm yea I went through that and scoured all the emails and pages the students see, still not seeing anything about submissions at all.

**Comment by nuke-web3** [comment]
+1 on this - reated: https://education.github.community/t/all-student-repositories-in-an-assignment-show-not-submitted/58568/10

It's *completely* unclear and undocumented behavior for ***YEARS** seems like a failure of this platform. Comeon Github :pray: .

![image](https://user-images.githubusercontent.com/35669742/205412389-c0747eeb-b24b-4fbf-96a7-b6e51151eae7.png)

I "deactivate" (is that just the invite link or more?) the assignment, and after the deadline is passed apparently arbitrary users are marked as submitted or not.

Perhaps related to some part of the CI grading completing or not? All tests are checked for everyone I have configured, so no idea.

**Reply by nuke-web3** [reply]
Maybe it was just that it was processing? No action on my part and now we have something sensible: 
![image](https://user-images.githubusercontent.com/35669742/205413386-649a3462-4b88-4536-a479-8d883f463a6b.png)
